pkgs/applications/version-management/gitui/default.nix

```diff
@@ -37,11 +37,19 @@ rustPlatform.buildRustPackage rec {
   # environment: delete them.
   postPatch = "rm .cargo/config";
 
+
+  # Getting app_config_path fails with a permission denied
+  checkFlags = [
+    "--skip=keys::key_config::tests::test_symbolic_links"
+  ];
+
+
   meta = with lib; {
     description = "Blazing fast terminal-ui for Git written in Rust";
     homepage = "https://github.com/extrawurst/gitui";
     changelog = "https://github.com/extrawurst/gitui/blob/${version}/CHANGELOG.md";
+    mainProgram = "gitui";
     license = licenses.mit;
-    maintainers = with maintainers; [ Br1ght0ne yanganto ];
+    maintainers = with maintainers; [ Br1ght0ne yanganto mfrw ];
   };
```

Comment:
Can you put this change in a dedicated commit please?

Reply:
Sure :) Will do